### ISSUE TITLE
Error when rendering component <md-button>, <md-sidebar>, ...

### ISSUE THREAD

**Issue by sk22** [marcosmoura/vue-material#285]
When trying to create a md-sidebar for instance, I get an error message.
Interestingly, this only happens when I'm using webpack, so I can't create a CodePen for it, since it works well there.

```
[Vue warn]: Error when rendering component <md-sidenav>:
```
```
TypeError: t._v is not a function
    at Proxy.render (vue-material.js:9)
    at VueComponent.Vue._render (vue.js:2927)
    at VueComponent.<anonymous> (vue.js:2335)
    at Watcher.get (vue.js:1643)
    at new Watcher (vue.js:1635)
    at VueComponent.Vue._mount (vue.js:2334)
    at VueComponent.Vue$3.$mount (vue.js:5916)
    at VueComponent.Vue$3.$mount (vue.js:8199)
    at init (vue.js:2648)
    at createComponent (vue.js:4030)
```

### Steps to reproduce
1. Set up a webpack project like mine: https://github.com/sk22/spg-tinf-sem03/tree/7fe21005b20a0fcee9f024b20e74b1fc7452a851/proj01
2. Uncomment the <md-sidenav> in `src/components/App.vue`
3. Run a webpack build
4. Open the project in a browser (`npm run dev` or open `index.html`)
5. Watch the error happen

### Versions
`vue-material@0.5.2`
`vue@2.1.4`
`webpack@2.2.0-rc.2`

**Comment by korylprince** [comment]
@sk22 I'm not seeing the issue you're posting. Here's what I did:

```
npm --version 
4.0.5
node --version
v7.3.0
git clone https://github.com/sk22/spg-tinf-sem03.git
cd spg-tinf-sem03/proj01
npm install
npm run build

```

The only console error I get is because you've got index.js referenced in src/index.html when it shouldn't be (because webpack inserts bundle.js into the html for you.)

Other than that the application builds and runs fine. If I add an md-button with `@click="$refs.sidenav.toggle()"` it works as expected.

My guess is that your node_modules may be borked. I'd try removing the whole directory and running npm install again. Or perhaps your node/npm version is too old/new. (I'm on arch linux, so I figure I've got the latest stable.)
